test/Tests/src/Semantic/Meta_Spec.enso

```diff
@@ -148,7 +153,7 @@ spec = Test.group "Meta-Value Manipulation" <|
 
     Test.specify "should allow to get the source location of a frame" pending=location_pending <|
         src = Meta.get_source_location 0
-        loc = "Meta_Spec.enso:150:15-40"
+        loc = "Meta_Spec.enso:155:15-40"
```

Comment:
Can we move this test to a separate file? Then we won't need to update it on each change.

Reply:
Done